modules/monitoring/application/views/scripts/partials/comment/comment-detail.phtml

```diff
@@ -1,17 +1,34 @@
 <div class="comment-author">
 <?php if ($comment->objecttype === 'service') {
-    echo $this->icon('service', $this->translate('Service'));
```

Comment:
Why is the icon removed? A host event still got an icon.

Reply:
I think the icon won't work as intended. Especially because, there isn't any consistency in other views, such as service~ and host~.

@theFeu: Of course, we should remove the icon consistently, namely downtime list as well.